internal/handler/handler.go

```diff
@@ -33,6 +33,81 @@ type userMgmtSvc struct {
 	logic logic.UserMgmtSvcLogicIer
 }
 
+func validatePass(pass string) *respModel.Response {
+	if len(pass) < 8 {
```

Comment:
The length check is handled via the `validate` struct tag. No need for this check.

Reply:
done